app/views/devise/mailer/confirmation_instructions.html.erb

```diff
@@ -0,0 +1,5 @@
+<p>Welcome <%= @email %>!</p>
+
+<p>You can confirm your account email through the link below:</p>
+
+<p><%= link_to 'Confirm my account', confirmation_url(@resource, confirmation_token: @token) %></p>
```

Comment:
- [ ] [REQUIRED] Good job so far! I appreciate what you have done but the email registered in the `"Login"` and `"Signup"` parts is not confirmed, if the registered email is not confirmed, it is not possible to go from the `"Login"` page to other pages. This does not allow you to see all the properties of this project, so please fix this error 🙏🏼 

![image](https://github.com/albertkantwi/recipe-app/assets/63915024/c8a22b4e-f25d-4219-89ae-fcc88fc1e6ab)